Missionframework/modules/00_init/README.md

```diff
@@ -0,0 +1,67 @@
+# KP Liberation Module Description
+
+## Init Module
+The init module handles the general mission initialization and distributes it to all clients.
+It contains the general/vital CBA settings for the mission, defines basic global variables and ui classes/values.
+Also the presets are loaded and checked for syntax errors. If any errors are found, it'll end the mission with a hint.
+Furthermore it takes care of the saving process by running scheduled savings.
+Mostly everything is processed on the server, which distributes the final data to the clients.
+
+### Dependencies
+This module uses no data or functions from other modules.
+
+### Functions
```

Comment:
The module and dependencies description is an great addition,  I'm not sure if functions needs to be described here. It adds another place where you need to document the function and it basically contains same stuff as `functions.hpp` comments.

Reply:
I added the functions "duplication" there with a wiki page generating process in mind, which is able to read these markdown files. But the site/wiki is your baby of course, so just tell me if we need/keep it or if I should remove it. 🙂

Reply:
Lets leave it for now.